less/home-page.less

```diff
@@ -25,6 +34,20 @@
       border-radius: 20px;
       height: 100%;
     }
+
```

Comment:
Next time you can set the pictures to display: none; Try using the not sudo class. &:not(:first-child){display: none;}